website/docs/how-to/customize-extensions/extension-editor.md

```diff
@@ -38,7 +38,7 @@ Let's break it down:
 
 * The identifier used when addressing your specific user
 
-**Ext**: `&ext=add-record`
+**Fn**: `&fn=add-record`
```

Comment:
We need to keep this as `Fn` because it is in the query params, still. We aren't changing that just yet.